intranet/static/js/eighth/statistics.js

```diff
@@ -0,0 +1,79 @@
+/* global $ */
+$(document).ready(function() {
+    if (Object.keys(raw_data).length > 0) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["raw_data" is not defined.](https://www.codacy.com/app/tjhsst-ion/ion/file/2623470290/issues/source?bid=3212528&fileBranchId=3299315#l3)